### ISSUE TITLE
[BUG] Cypress: Configuration not loading

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#274]
## What tool(s) are you using?
* cypress-accessibility-checker

## Detailed description
assertCompliance was returning a "2" for failLevels: ["violation"]. When Nick added it to the cypress.json env, it worked:
```
{
  "env": {
    "failLevels": ["violation"]
  }
}
```

Must be checking as a command. We need to figure out how to get the config to load from the files.
